tests/Fixtures/Integration/set/@PHP8Migration.test

```diff
@@ -1,6 +0,0 @@
---TEST--
```

Comment:
Actually this is not from a commit of yours, maybe you need to rebase?

Reply:
Yes, this is because this PR is WIP and contains commits from my other PR joint with @SpacePossum. These will disappear after that is merged and this PR is rebased. ;)